data_gateway/cli.py

```diff
@@ -45,11 +45,15 @@ def gateway_cli(logger_uri, log_level):
     from octue.log_handlers import apply_log_handler, get_remote_handler
 
     # Apply log handler locally.
-    apply_log_handler(log_level=log_level.upper())
+    apply_log_handler(log_level=log_level.upper(), include_thread_name=True)
```

Comment:
What happens to the log if we shift to include_thread_name=true by default? Does it get really messy?

Maybe we should simplify down to have a `detail=true` option that includes threads, process ids, the whole lot.

Reply:
It's just more stuff to fit on to one line. I find that logs are much more readable if each log entry is always on one line, otherwise it really disrupts the flow of reading them as a human. It also seems not that relevant to provide a process and/or thread name for single-threaded/single-process programs, which I imagine most of the apps using `octue-sdk-python` will be.

We could simplify to `detail=True` but then if you're just multi-threading or just multi-processing, you get other non-useful information in the log entry.

Reply:
I think no need to act now, just toying with clearer / more compact ways of wrapping things up.